gode/packages.go

```diff
@@ -101,8 +110,7 @@ func execNpm(args ...string) (string, string, error) {
 }
 
 func environ() []string {
-	env := append(os.Environ(), "NPM_CONFIG_SPIN=false")
```

Comment:
Notably this doesn't actually do anything in npm 3, this was the setting for npm 2. The reason that the spinner wasn't showing was because stdout wasn't a tty (though the spinner shows on stderr)